# lidar_topic: "/velodyne_points"
lidar_topic: "/lslidar_point_cloud"
ground_topic: "/cloud/ground_cloud"
no_ground_topic: "/cloud/no_ground_cloud"

is_downSample: false
voxel_leaf_size: 1.0

is_clipRadius: true
minClipRadius: 2.0
maxClipRadius: 70.0

is_clipHeight: true
minClipHeight: 1.0

maxFloorAngle: 0.5    # rad : the maximum allowed difference between the model normal and the given axis in radians.
maxFloorHeight: 0.10    # m